### ISSUE TITLE
Why to use "g_runing" and "accumulate"?

### ISSUE THREAD

**Issue by RyanYChen** [rosinality/style-based-gan-pytorch#29]
Hi! I'm new in GAN's field. I have a question about why to use "g_runing" and "accumulate" to do a weighted-average to generator's parameter during training? And why to use "g_runing" instead of trained generator during testing?   

Is this a trick during GAN's training? Or does this appear in any paper?

Thanks a lot.

**Comment by rosinality** [comment]
Yes, it appears in progressive gan and style gan paper, and widely used in gan training (like biggan.) It is effective for model stability and sample quality.

**Comment by RyanYChen** [comment]
> Yes, it appears in the progressive gan and style gan paper, and widely used in gan training (like biggan.) It is effective for model stability and sample quality. Using moving average of parameters is often useful for performance of neural nets.

Thanks a lot!
